internal/app/orchestrator/orchestrator_test.go

```diff
@@ -55,9 +56,9 @@ func (m mockMultiStreamUpstreamClient) OpenStream(
 
 func newMockOrchestrator(t *testing.T, mockScope tally.Scope, mapper mapper.Mapper,
 	upstreamClient upstream.Client) *orchestrator {
-	logger := log.New("info")
+	logger := log.New("info", os.Stderr)
 	orchestrator := &orchestrator{
-		logger:                logger,
+		logger:                log.New("info", os.Stderr),
```

Comment:
is it worth creating a mocklogger or setting these log levels and stderr as default for testing/no config field provided?

Reply:
that's a good idea, I can do it as a part of this PR.